feature.h

```diff
@@ -12,22 +12,23 @@
 
 #define HINT_FEATURE_SHIFT	26
 
-#define FEATURE_BITWISE_BIT          0x0001
-#define FEATURE___SUB___BIT          0x0002
-#define FEATURE_MYREF_BIT            0x0004
-#define FEATURE_EVALBYTES_BIT        0x0008
-#define FEATURE_FC_BIT               0x0010
-#define FEATURE_INDIRECT_BIT         0x0020
-#define FEATURE_ISA_BIT              0x0040
-#define FEATURE_MULTIDIMENSIONAL_BIT 0x0080
-#define FEATURE_POSTDEREF_QQ_BIT     0x0100
-#define FEATURE_REFALIASING_BIT      0x0200
-#define FEATURE_SAY_BIT              0x0400
-#define FEATURE_SIGNATURES_BIT       0x0800
-#define FEATURE_STATE_BIT            0x1000
-#define FEATURE_SWITCH_BIT           0x2000
-#define FEATURE_UNIEVAL_BIT          0x4000
-#define FEATURE_UNICODE_BIT          0x8000
+#define FEATURE_BAREWORD_FILEHANDLES_BIT 0x0001
```

Comment:
Is there a reason why this new feature bit has to be shoved in before all the others, thereby causing a general re-numbering?

Reply:
Preserving alphabetical ordering.

Reply:
The file (and specifically the ordering) is generated.